perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -55,23 +58,71 @@ sub bin_services_for_address {
     };
 
     my $site_services = $self->whitespace->GetSiteCollections($property->{uprn});
-
-    # TODO: Filter out services with SiteServiceValidTo in the past or SiteServiceValidFrom in the future
-    $site_services = [ grep { $_->{NextCollectionDate} } @$site_services ];
-
-    my $services = [ map {
-        {
-            id => $_->{SiteServiceID},
-            service_name => $_->{ServiceItemDescription},
-            next => {
-                date => $_->{NextCollectionDate},
-                ordinal => 'th', # TODO!
-                changed => 0,
-            },
+    my @site_services_filtered;
+
+    my $now_dt = DateTime->now->set_time_zone( FixMyStreet->local_time_zone );
+    for my $service (@$site_services) {
+        # TODO Do we need to filter out services where NextCollectionDate is
+        # in the past?
```

Comment:
From what I've seen these dates are either in the future or blank, so don't think we need to worry about dates in the past at this point.